testing/ltp/Make.defs

```diff
@@ -0,0 +1,39 @@
+############################################################################
+# apps/testing/Make.defs
```

Comment:
replace the copyright

Reply:
@xiaoxiang781216 update done

Reply:
Strange, it still displaying BSD License instead of Apache, is it some issue on github?

Reply:
@acassis Thanks for your reminder, update done.